### DISCUSSION TITLE
请问官网下载的maa压缩包下载到哪里去了？

### DISCUSSION THREAD

**Discussion by bigbrother001** [MaaAssistantArknights/MaaAssistantArknights#4484] [Q&A]
我在maa.plus点击windowsx64下载，结果下载完后到处都找不到下载下来的东西在哪。下载那个文件夹下没有，浏览器的下载记录里也找不到下载的文件。所以请问到底下载到哪里去了？

**Comment by MistEO** [comment]
@GalvinGao 要不，，再整个直接下载的链接（？

**Reply by quark-mac** [reply]
搞个直接下的吧，我当初下载MAA也是这样😂

**Comment by GalvinGao** [comment]
请问使用的浏览器是什么呢

**Reply by bigbrother001** [reply]
edge浏览器

**Reply by GalvinGao** [reply]
ummm，具体是什么版本呢
可以前往 https://ping.huatuo.qq.com 仅复制 “以下是您操作系统相关信息” 部分到这里

**Reply by bigbrother001** [reply]
操作系统：Win32浏览器：chrome:112

用户代理：Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36 Edg/112.0.1722.68

Flash版本：未安装Cookie状态：开启

JavaScript状态：开启LocalStorage状态：开启


是否联网：是网络类型：unknown

有效带宽：unkown有效RTT：150 ms
